cli/tests/integration/coverage_tests.rs

```diff
@@ -575,3 +590,42 @@ fn test_html_reporter() {
       .unwrap();
   assert!(baz_quux_ts_html.contains("<h1>Coverage report for baz/quux.ts</h1>"));
 }
+
+#[test]
+fn test_summary_reporter() {
+  let context = TestContext::default();
+  let tempdir = context.temp_dir();
+  let tempdir = tempdir.path().join("cov");
+
+  let output = context
+    .new_command()
+    .args_vec(vec![
+      "test".to_string(),
+      "--quiet".to_string(),
+      format!("--coverage={}", tempdir),
+      "coverage/multisource".to_string(),
+    ])
+    .run();
+
+  output.assert_exit_code(0);
+  output.skip_output_check();
+
+  let output = context
+    .new_command()
+    .args_vec(vec!["coverage".to_string(), format!("{}/", tempdir)])
+    .run();
+
+  output.assert_exit_code(0);
+  output.assert_matches_text(
+    "----------------------------------
+File         | Branch % | Line % |
+----------------------------------
+ All files   |    40.00 |  60.98 |
```

Comment:
Does this output match one from any of the existing tools?

I'm thinking that maybe we could do something like this:
```
----------------------------------
File         | Branch % | Line % |
----------------------------------
 bar.ts      |     0.00 |  57.14 |
 baz/quux.ts |     0.00 |  28.57 |
 baz/qux.ts  |   100.00 | 100.00 |
 foo.ts      |    50.00 |  76.92 |
----------------------------------
 All files   |    40.00 |  60.98 |
----------------------------------
```

Reply:
Updated. Looks better to me

<img width="601" alt="Screenshot 2023-12-12 at 0 15 45" src="https://github.com/denoland/deno/assets/613956/a4df1a50-5a83-4c6b-856b-bd4f1f6a9ada">

Reply:
Probably one decimal place of precision would be enough.

Reply:
Updated

<img width="584" alt="Screenshot 2023-12-12 at 1 05 34" src="https://github.com/denoland/deno/assets/613956/f9eb012c-a83b-4d66-883f-4f86fe646a07">